Added byte sequence shaper to churn

blanu opened the pull request:
Added byte sequence shaper config to churn
Added byte sequence shaper constructor to churn-pipe
Some minor reformatting.

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/uproxy/uproxy-lib/282)
<!-- Reviewable:end -->

trevj commented on src/churn/churn.ts:
```diff
@@ -301,6 +301,19 @@ export var filterCandidatesFromSdp = (sdp:string) : string => {
       //       'key': '0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0'
       //     }));
+
+      /*this.pipe_.setTransformer('byteSequenceShaper', undefined,
```
Why don't you follow the same convention as AES, i.e. C++-style comments, with our regular formatting.

blanu commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
Okay I have made this change.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->

trevj commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions, some commit checks pending.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
And formatting? I meant if you uncomment this that it has roughly the same formatting as the other statements in this block.

---

<sup>**[src/churn/churn.ts, line 311 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysiK5IZjYgghpN4SZ0)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L311)):</sup>
spaces before and after `=`

---

<sup>**[src/churn/churn.ts, line 313 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-Jysi70R2g63U6MraJlp)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L313)):</sup>
Please roughly follow the style guide, i.e.:
```typescript
var sequence = {
  index: 0,
  offset: 0,
  sequence: hex,
  length: 256
};
```

---

<sup>**[src/churn/churn.ts, line 317 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysiHyUq1Q66CH86LU3)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L317)):</sup>
And here.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->

blanu commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions, all commit checks successful.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
Okay I have changed all of the initializer code to be consistent.

---

<sup>**[src/churn/churn.ts, line 311 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysiK5IZjYgghpN4SZ0)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L311)):</sup>
Okay I have made this change.

---

<sup>**[src/churn/churn.ts, line 313 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-Jysi70R2g63U6MraJlp)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L313)):</sup>
I have made this change.

However, this is not in the style guide. Therefore, I updated the style guide to include this.

---

<sup>**[src/churn/churn.ts, line 317 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysiHyUq1Q66CH86LU3)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/361800a2b73a04b3f632d7b1b9680adf0f03f6b6/src/churn/churn.ts#L317)):</sup>
Okay I have made this change.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->

trevj commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions, some commit checks pending.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
Did you forget to push your changes?

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->

blanu commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
I did push the changes. It is possible that you saw my new comments before my new changes.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->

trevj commented:
:+1:

---

Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

<sup>**[src/churn/churn.ts, line 305 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282#-JysfjaQP_j6zMiBbeI0-r1-305)** ([raw file](https://github.com/uproxy/uproxy-lib/blob/21823f246889d422e5d5ddc4cf5acafa601b94c5/src/churn/churn.ts#L305)):</sup>
Sorry, I think reviewable was being weird :-/

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-lib/282)
<!-- Sent from Reviewable.io -->
